sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -11,7 +11,15 @@ $.ajaxSetup({
   cache: false
 });
 
-
+var errorDecider = function (data, status, headers, config) {
+  switch(status) {
+    case 403:
+      window.location = '/sectionsUtilityTool/error.html';
+      break;
+    default:
+      //other errors
```

Comment:
No, actually. Even 500 we plan on putting one same error page. Probably might put up slightly different error message